lib/evss/reference_data/configuration.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+module EVSS
+  module ReferenceData
+    class Configuration < EVSS::Configuration
+      PROXY_OPTS = {
+        proxy: {
+          uri:  URI.parse(Settings.faraday_socks_proxy.uri),
+          socks: true
+        }
+      }.freeze
+
+      def base_path
+        # TODO: integrate with Settings.yml & devops
+        'https://internal-staging-services-1341723990.us-gov-west-1.elb.amazonaws.com/api/refdata/v1'
+      end
+
+      def service_name
+        'EVSS/ReferenceData'
+      end
+
+      def mock_enabled?
+        # TODO: create mock data
+        false
+      end
+
+      def client_cert
+        # TODO : implement
```

Comment:
Are client cert/key needed in JWT world?

Reply:
The JWT is signed but not encrypted so I'd assume so.  I re-read over the design doc `Vets.gov EVSS Trust Relationship v2` and it's ambiguous.  TBD until I can speak with EVSS on the matter.